testsuites/listener/shared/performance/conftest.py

```diff
@@ -0,0 +1,184 @@
+    parser.addoption("--liteserv-port", action="store", help="liteserv-port: the port to assign to liteserv")
+    parser.addoption("--liteserv-storage-engine", action="store", help="liteserv-storage-engine: the storage engine to use with liteserv")
+    parser.addoption("--skip-provisioning", action="store_true", help="Skip cluster provisioning at setup", default=False)
+    parser.addoption("--sync-gateway-version", action="store", help="sync-gateway-version: the version of sync_gateway to run tests against")
+    parser.addoption("--sync-gateway-mode", action="store", help="sync-gateway-mode: the mode of sync_gateway to run tests against, channel_cache ('cc') or distributed_index ('di')")
+    parser.addoption("--server-version", action="store", help="server-version: version of Couchbase Server to install and run tests against")
+    parser.addoption("--xattrs", action="store_true", help="Use xattrs for sync meta storage. Sync Gateway 1.5.0+ and Couchbase Server 5.0+")
+    parser.addoption("--device", action="store_true", help="Enable device if you want to run it on device")
+
+
+# This will get called once before the first test that
+# runs with this as input parameters in this file
+# This setup will be called once for all tests in the
+# testsuites/listener/shared/client_sg/ directory
+@pytest.fixture(scope="session")
+def setup_client_syncgateway_suite(request):
+
+    """Suite setup fixture for client sync_gateway tests"""
+
+    log_info("Setting up client sync_gateway suite ...")
+
+    liteserv_platform = request.config.getoption("--liteserv-platform")
+    liteserv_version = request.config.getoption("--liteserv-version")
+    liteserv_host = request.config.getoption("--liteserv-host")
+    liteserv_port = request.config.getoption("--liteserv-port")
+    liteserv_storage_engine = request.config.getoption("--liteserv-storage-engine")
+
+    skip_provisioning = request.config.getoption("--skip-provisioning")
+    sync_gateway_version = request.config.getoption("--sync-gateway-version")
+    sync_gateway_mode = request.config.getoption("--sync-gateway-mode")
+
+    server_version = request.config.getoption("--server-version")
+    xattrs_enabled = request.config.getoption("--xattrs")
+    device_enabled = request.config.getoption("--device")
+
+    liteserv = LiteServFactory.create(platform=liteserv_platform,
+                                      version_build=liteserv_version,
+                                      host=liteserv_host,
+                                      port=liteserv_port,
+                                      storage_engine=liteserv_storage_engine)
+
+    if xattrs_enabled and version_is_binary(sync_gateway_version):
+            check_xattr_support(server_version, sync_gateway_version)
+
+    log_info("Downloading LiteServ ...")
+    # Download LiteServ
+    liteserv.download()
+
+    # Install LiteServ
+    if device_enabled and liteserv_platform == "ios":
```

Comment:
what about android?

Reply:
no need for android as there is not special commands needed for device . Only for iOS